Sources/CSRF/CSRF.swift

```diff
@@ -42,30 +42,29 @@ public struct CSRF: Middleware {
     /// Creates a token from a given `Request`. Call this method to generate a CSRF token to assign to your key of choice in the header and pass the token back to the caller via the response.
     /// - parameter request: The `Request` used to either find the secret in, or the request used to generate the secret.
     /// - returns: `Bytes` representing the generated token.
```

Comment:
This no longer returns `Bytes`. Can you change this to `String`?

Reply:
Wanted to bump this change in case you missed it.

Reply:
Didn't miss it, but for some reason, Github doesn't show that change… just check the full diff.